Tween/TimedAnimation.cs

```diff
@@ -68,8 +56,8 @@ public float Duration
 
 		public override bool IsValid { get { return TargetObject != null || TargetRectTransform != null; } }
 
-		protected bool IsComplete { get { return IsReversed ? Progress <= 0 : Progress >= 1.0f; } }
-		protected float Progress { get { return Mathf.Clamp01(CurrentTime / Duration); } }
+		protected bool IsComplete { get { return IsReversed ? Progress <= 0f : Progress >= 1f; } }
+		private float Progress { get { return Mathf.Clamp01(CurrentTime / Duration); } }
```

Comment:
Is there any reason not to make this public ?

Reply:
Yes.
1. It was protected and nobody was using it;
2. `Duration` can be 0 and you will have a "divide by 0" issue -- making it private will decrease the chance spreading this issue.